code/table_1.R

```diff
@@ -27,21 +27,56 @@ style_file(paste(gbv_project_wd, "/code/table_1.R", sep = ""))
 path_to_clean_rds <- paste(gbv_project_wd, "/data/clean/demographic_data_clean.RDS", sep = "")
 clean_data <- readRDS(path_to_clean_rds)
 
-# Create table
+# Create table 1 - for publication
 filtered_data <-
   clean_data %>%
   filter(status == "All three") %>%
   mutate(position_groups = droplevels(position_groups))
 
 demographic_table <- filtered_data %>%
-  select(c("sex_factored", "age_groups", "position_groups", "position_years_clean", "municipality")) %>%
+  select(c(
+    "sex_factored", "age_groups", "position_groups", "position_years_clean",
+    "municipality", "previous_training_factored"
+  )) %>%
   tbl_summary(by = municipality, label = list(
     sex_factored ~ "Sex",
     age_groups ~ "Age (years)",
     position_groups ~ "Position",
-    position_years_clean ~ "Years of practice"
+    position_years_clean ~ "Years of practice",
+    previous_training_factored ~ "Previous GBV Training"
   ), type = list(position_years_clean ~ "continuous")) %>%
   add_overall() %>%
   add_n()
 
 demographic_table
+
+# Create table 1 - for comparison to <2 timepoints
+# create new variable for <2 timepoints
+clean_data <- clean_data %>%
+  mutate(timepoints = case_when(
+    status == "All three" ~ "three",
+    TRUE ~ "<three"
+  ))
+
+filtered_data_comparison <-
+  clean_data %>%
+  mutate(position_groups = droplevels(position_groups))
+
+demographic_table_comparison <- filtered_data_comparison %>%
+  select(c(
+    "sex_factored", "age_groups", "position_groups", "position_years_clean",
+    "municipality", "timepoints", "previous_training_factored"
+  )) %>%
+  tbl_summary(by = timepoints, label = list(
```

Comment:
Because we standardized all of the demographic variables across time points, are you expecting to see differences by timepoint here? There shouldn't be a difference by timepoint here.

Reply:
I see. I think timepoint is a confusing name for that variable because we have a variable called timepoint that is 1, 2, or 3. I'd rename this variable to something like `status_binary`.